scripts/run-integ-test

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+"""Run Integ Tests based on the changed files
+
+"""
+from subprocess import call, check_call, Popen, PIPE
+
+# Minimal modules to tests when core changes are detected.
+# s3 - xml, dynamodb - json, sqs - query
+
+minimum_modules_to_test = ["s3", "dynamodb", "sqs"]
+
+def check_diffs():
+    """
+    Retrieve the changed files
+    """
+
+    #  git diff HEAD~ HEAD --name-only
+    ret = Popen(["git", "diff", "HEAD~", "HEAD", "--name-only"], stdout=PIPE)
+
+    diff, stderr = ret.communicate()
+    return diff.splitlines(False)
+
+def get_modules(file_path):
+    """
+    Parse the changed file path and get the respective module names
+    """
+    path = file_path.split('/')
+    top_directory = path[0]
+
+    if top_directory in ["core", "http-clients", "codegen"]:
+        return minimum_modules_to_test
+    elif top_directory== "services":
+        return path[1]
+
```

Comment:
There's no `else` clause so this can return `None`. Is that expected?

Reply:
Yes, this is expected. We don't need to run integ tests for changes in other modules such as `build-tools`